src/BizHawk.Client.EmuHawk/MainForm.Events.cs

```diff
@@ -2707,7 +2707,21 @@ private static void FormDragEnter(object sender, DragEventArgs e)
 		}
 
 		private void FormDragDrop(object sender, DragEventArgs e)
-			=> PathsFromDragDrop = (string[]) e.Data.GetData(DataFormats.FileDrop);
+			=> this.BeginInvoke(((Action<DragEventArgs>)(args =>
+			{
+				var pathsFromDragDrop = (string[])args.Data.GetData(DataFormats.FileDrop);
+				if (pathsFromDragDrop is not null) this.DoWithTempMute(() =>

```

Comment:
This was moved out of the drop event handler because of #1483—does this commit reintroduce that bug?